notebooks/fixtures/template/translate-template.yaml

```
@@ -0,0 +1,3 @@
+flowers:
+  roses: @[ color1 ]
+  violets: @[ color2 ]
```

I am not sure that atparse would allow the spaces between the `[]`. Making this change would change would drive a change in `translate-complete.yaml` if I'm not mistaken.

```suggestion
  roses: @[color1]
  violets: @[color2]
```

---

Thanks! I updated to reflect the changes here and to `translate-complete.yaml` as well.